Makefile

```diff
@@ -1,10 +1,7 @@
-test: flake8 pylint pytest
+test: static pytest
 
-flake8:
-	flake8 nameko_bayeux_client tests
-
-pylint:
-	pylint nameko_bayeux_client -E
```

Comment:
Intentionally dropping pylint and not replacing with a pre-commit check?

Reply:
Just slipped away! .)

Added back as a pre-commit (and local) entry.